content/vendor/guides/ci-cd-integration.md

```diff
@@ -0,0 +1,134 @@
+---
+date: "2020-02-20T00:00:00Z"
+lastmod: "2020-02-20T00:00:00Z"
+title: "CI/CD Integration"
+weight: "1007"
+---
+
+The KOTS CI/CD Integration guide is an advanced workflow that allows you to use a similar workflow to [Kots Starter GitHub Actions](https://github.com/replicatedhq/replicated-starter-kots/blob/master/.github/workflows/main.yml#L21) but for your own internal git repository and CI/CD systems like GitHub, GitLab, CircleCI, TravisCI, Jenkins, etc.
+
+It is broken into three sections:
+
+- [Environment Variables](#environment-variables)
+- [Kubernetes Objects](#kubernetes-objects)
+- [Helm Chart](#helm-chart)
+
+## Environment Variables
+
+This section will assume you've already completed the [Automating Your Workflow](/vendor/guides/quickstart/#automating-your-workflow) and set up a local git repo to manage your application.
+
+- Configure the following 2 environment variables in your CI/CD configuration.
+
+- `REPLICATED_APP` with the App Slug.
+- `REPLICATED_API_TOKEN` with the API token.
+
+## Kubernetes Objects
+
+Follow the steps below if you are just working with plain-YAML kubernetes API objects.
+
+- In the root directory of the internal repository add this `Makefile`
+
+```shell
+curl -LS https://raw.githubusercontent.com/replicatedhq/replicated-starter-kots/master/Makefile -o ./Makefile
+```
+
+- Make a copy of [manifests](https://github.com/replicatedhq/replicated-starter-kots/tree/master/manifests) directory from the KOTS starter repo and place it in your internal repo's root directory.
```

Comment:
what if I already have my manifests in another folder like `k8s` or `deploy` -- does this mean I have to re-organize my repo in order to make it work with kots?

Reply:
I like that the helm chart section of this guide started with an example repo structure *before* adding the kots bits -- what if we did the same here? Maybe we could assume the manifests start out in a non-standard place (i.e. not `manifests`)

Reply:
Good idea, updated.